love/components/filters.tsx

```diff
@@ -31,12 +36,33 @@ const initialFilters = {
 export const Filters = (props: {
   allLovers: Lover[] | undefined
   setLovers: (lovers: Lover[] | undefined) => void
+  youLover: Lover | undefined | null
 }) => {
-  const { allLovers, setLovers } = props
+  const { allLovers, setLovers, youLover } = props
   const [filters, setFilters] = usePersistentInMemoryState<
     Partial<rowFor<'lovers'> & User>
   >(initialFilters, 'profile-filters')
 
+  const [nearbyOriginLocation, setNearbyOriginLocation] = useState<
+    string | null | undefined
+  >(undefined)
+
+  const [radius, setRadius] = useState<number>(100)
+  const [debouncedRadius, setDebouncedRadius] = useState(radius)
+  const debouncedSetRadius = debounce(setDebouncedRadius, 200)
```

Comment:
I think this should be:

`const [debouncedSetRadius] = useState(() => debounce(setDebouncedRadius, 200))`

So that the function is not recreated on each render